components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/store/UserRecoveryDataStore.java

```diff
@@ -37,6 +38,21 @@ UserRecoveryData load(String code) throws
     UserRecoveryData load(User user) throws
             IdentityRecoveryException;
 
+    /**
+     * Load UserRecoveryData from the
```

Comment:
I think we might have missed some of the details here

Reply:
yes, added. Thanks for pointing out.